bin/import.js

```diff
@@ -1,5 +1,7 @@
 var fs = require('fs')
 var pump = require('pump')
+var url = require('url')
```

Comment:
minor nit, but I try and put all core requires at the top (so this should go below `fs`)